tap_twitter_ads/streams.py

```diff
@@ -206,7 +207,35 @@ def remove_hours_local(self, dttm, timezone):
         new_dttm = dttm.astimezone(timezone).replace(
             hour=0, minute=0, second=0, microsecond=0)
         return new_dttm
-       
+
+    def return_bookmark(self, bookmark_value_str, datetime_format, record_dict, bookmark_field, stream_name, record_counter, last_dttm):
```

Comment:
What is the difference between return_bookmark and get_bookmark? Can we rename it appropriately?

Reply:
`get_bookmark` function returns the bookmark value that is available in the state whereas `return_bookmark` calculates the maximum bookmark value by comparing the replication key value that is available in the records and returns it. Updated name of `return_bookmark` to `return_maximum_bookmark`.